drivers/sensor/iis2mdc/iis2mdc_spi.c

```diff
@@ -50,31 +49,26 @@ static int iis2mdc_spi_read(struct iis2mdc_data *data, uint8_t reg_addr,
 		.count = 2
 	};
 
-
-	if (len > 64) {
-		return -EIO;
-	}
```

Comment:
Did you mean to remove this check?

Reply:
Yes, because it should be not there. But, as already discussed with @pabigot in another PR, this will be done in a separate PR for all ST sensor drivers later on. So, I leave it in here.

Reply:
I mean, I'll leave the check in. Thanks again for catching it.